### PR TITLE
day 3 twoCityScheduling solution leetcode

### PR DESCRIPTION
**Pull request by humblefool96**

### CODE REVIEW

File: src/codes/june_challenge/day3/TwoCityScheduling.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        return a.second < b.second;
+    }
+    
+    int twoCitySchedCost(vector<vector<int>>& costs) {
+        int n = costs.size();
+        vector<pair<pair<int, int>, int>> vect;
+        for (int i = 0; i < n; i++) {
+            int diff = costs[i][0] - costs[i][1];
+            vect.push_back({{costs[i][0], costs[i][1]}, diff});
+        }
```
Comment (sharoha) [inline src/codes/june_challenge/day3/TwoCityScheduling.cpp]: You can sort the `costs` array without creating a temp vector

Comment (sharoha) [reply src/codes/june_challenge/day3/TwoCityScheduling.cpp]: Maybe use comparator directly with lambda?

**Review by sharoha — COMMENTED** [review]
Left a few comments.

File: src/codes/june_challenge/day3/TwoCityScheduling.cpp (hunk 1 above)
Comment (humblefool96) [reply src/codes/june_challenge/day3/TwoCityScheduling.cpp]: Seems like a nice idea, will do it.

**Review by sharoha — APPROVED** [review]
Thanks for the update

**Pull request merged**